sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_builtin_evaluators.py

```diff
@@ -171,7 +171,7 @@ def test_quality_evaluator_prompt_based_with_dict_input(self, model_config):
         assert score["gpt_fluency"] > 0.0
 
     def test_content_safety_evaluator_violence(self, project_scope, azure_cred, simple_conversation):
-        eval_fn = ViolenceEvaluator(project_scope, azure_cred)
+        eval_fn = ViolenceEvaluator(azure_cred, project_scope)
```

Comment:
NIT: maybe we should pass them as kwargs if the order change can break people's code?

Reply:
it's a good point-- I think it makes sense if we decide to expand what's passed here beyond just these 2 things. Will keep it as is for now though